pkgs/by-name/za/zapzap/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zapzap";
+  version = "4.5.5.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "zapzap-linux";
+    repo = "zapzap";
+    rev = version;
+    hash = "sha256-8IeFGTI+5kbeFGqH5DpHCY8pqzGhE48hPCEIKIe7jAM=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    dbus-python
+    pyqt6
+    pyqt6-webengine
+    pyqt6-sip
+  ];
+
+  prePatch = "export HOME=$(mktemp -d)";
+
```

Comment:
```suggestion
```

Reply:
I can not build without this.

Reply:
I've moved out above to the correct phase because we don't need it that early and generally prePatch should only prepare files to apply patches.